src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,51 @@
+$mainColor: #00acdc;
+
+.nav {
+  &__list {
+    display: flex;
+    align-items: center;
+    list-style-type: none;
+    padding: 0;
+    margin: 0;
+  }
+
+  &__item {
+    margin-left: 20px;
+  }
+
+  &__item:first-child {
+    margin-left: 0;
+  }
+
+  &__link {
+    display: flex;
+    align-items: center;
+    height: $headerHight;
```

Comment:
There seems to be a typo in the variable name `$headerHight`. It should likely be `$headerHeight` to correctly represent 'height'. Please check the variable definition and usage.